src/main/java/io/turntabl/dataformatting/TCMS.java

```diff
@@ -0,0 +1,20 @@
+package io.turntabl.dataformatting;
+
+public class TCMS {
+    public static void main(String[] args) {
+       // ClientDetails clientDetails = new ClientDetails("marfo kofi","kkk","sdfghjkl");
```

Comment:
Can't you move all this printing to a void method in a package, so that we call this method in main later

Reply:
i tried but it was not printing in the format so i was looking out how best we can do it